### PR TITLE
Marine Radio Channels

### LINKED ISSUE #210: add radio channels

### PR DESCRIPTION
**Pull request by Tunguso4ka**
## About the PR
Adds telecomms and next marine radio channels from CM13:

```
frequency - keycode - name

1471 - z - HighCommand

1480 - g - Marine Common 
1481 - v - Command
1482 - m - Medical
1483 - n - Engineer
1484 - p - Military Police
1485 - u - Requisition
1486 - t - Intel

1491 - a - Alpha
1492 - b - Bravo
1493 - c - Charlie
1494 - d - Delta
1495 - e - Echo
1496 - f - Foxtrot

1381 - o - Colony
```

To do:
- [x] Make channels work
- [x] Encryption keys (sprites and prototypes)
- [x] Radio headsets (sprites and prototypes)
- [x] Telecomms server (sprites and prototypes)
- [x] Make ghosts hear channels

## Why / Balance
Resolves #210 

## Media

image_2023-11-20_17-35-07

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

### CODE REVIEW

**Comment by Tunguso4ka** [comment]
don't know any solution that isn't requires upstream code/yml change. So closed for the best times.

**Comment by DrSmugleaf** [comment]
@Tunguso4ka For radio channels it's probably fine to modify upstream files, I can try to figure out something else later but feel free to reopen the PR

**Comment by Tunguso4ka** [comment]
> For radio channels it's probably fine to modify upstream files

well, removing keys from Res../Pro../radio_channels.yml triggers fatal error. I have a simple stupid idea to fix that by changing upstream radio keys for random characters. For example Cyrillic chars.

**Comment by Tunguso4ka** [comment]
Stupid decisions makes billions.

**Comment by QuietlyWhisper** [comment]
The easiest fix was to make upstream radios russian amazing

**Comment by QuietlyWhisper** [comment]
Everything looks fine so far, you can port just the ones needed for first playtest if you want to save time and do the rest after

**Comment by Tunguso4ka** [comment]
> The easiest fix was to make upstream radios russian amazing

well, to be fair its more Ukrainian/Belarusian because of i, but okay.

**Comment by lapatison** [comment]
Can ghosts hear the new channels?

**Comment by Tunguso4ka** [comment]
> Can ghosts hear the new channels?

that`s good that you asked about that. Because if you not, then the answer is no.

**Comment by Tunguso4ka** [comment]
okay, the channels are work, except of marine common. With is fucking ":;". Techiclly it works, but not how it should.

**Comment by Tunguso4ka** [comment]
ComputeManagedAssemblies task not found. Meh

**Comment by Tunguso4ka** [comment]
Ready to review.

File: Resources/Prototypes/_CM14/Entities/Objects/encryption_keys.yml [hunk 1]
```diff
@@ -0,0 +1,395 @@
...
+    - MarineIntel
+    defaultChannel: MarineIntel
+  - type: Sprite
+    state: cap_key
+
+# Vehicle Crewman
+- type: entity
+  parent: CMEncryptionKey
+  id: CMEncryptionKeyCrewman
+  name: vehicle crewman`s encryption Key
```
Comment (QuietlyWhisper) [inline Resources/Prototypes/_CM14/Entities/Objects/encryption_keys.yml:367]:
```suggestion
  name: vehicle crewman`s encryption Key
  noSpawn: true # We might not need this
```

**Review by QuietlyWhisper — APPROVED** [review]

**Pull request merged**